app/views/smart_answers/_previous_answers.html.erb

```diff
@@ -13,7 +13,13 @@
     </thead>
     <tbody class="govuk-table__body">
       <% @presenter.collapsed_questions.each_with_index do |question, number_questions_so_far| %>
-        <%= render partial: 'previous_answer', locals: {question: question, number_questions_so_far: number_questions_so_far} %>
```

Comment:
Just to confirm, as it's not mentioned in the commit: we're intentionally losing the `number_questions_so_far` local? It does appear to be unused in the template.

Reply:
Yup! To make it consistent we are passing in all the required attributes from `@presenter`.